piggy/1.5/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM staphb/roary:3.12.0
+
+# metadata
+LABEL base.image="staphb/roary:3.12.0"
+LABEL version="1"
```

Comment:
please change line 5 to `LABEL dockerfile.version="1"`

explanation here: https://github.com/StaPH-B/docker-builds/issues/118